src/commands/doc.rs

```diff
@@ -0,0 +1,112 @@
+//! Implementation of the `doc` command.
+
+use std::path::PathBuf;
+
+use anyhow::Context;
+use anyhow::Result;
+use anyhow::bail;
+use clap::Parser;
+use wdl::doc::build_stylesheet;
+use wdl::doc::build_web_components;
+use wdl::doc::document_workspace;
+use wdl::doc::install_theme;
+
+/// Arguments for the `doc` subcommand.
+#[derive(Parser, Debug)]
+#[command(author, version, about)]
+pub struct Args {
+    /// Path to the local WDL workspace to document.
+    pub workspace: PathBuf,
+    /// Path to a Markdown file to embed in the `<output>/index.html` file.
+    #[arg(long, value_name = "MARKDOWN FILE")]
+    pub homepage: Option<PathBuf>,
+    /// Output directory for the generated documentation.
+    /// If not specified, the documentation will be generated in
+    /// `<workspace>/docs`.
+    #[arg(long, value_name = "DIR")]
+    pub output: Option<PathBuf>,
+    /// Overwrite any existing documentation.
+    ///
+    /// If specified, any existing files in the output directory will be
+    /// deleted. Otherwise, the command will ignore existing files.
+    /// Regardless of this flag, the command will overwrite any existing
+    /// files which conflict with the generated documentation.
+    #[arg(long)]
+    pub overwrite: bool,
+    /// Open the generated documentation in the default web browser.
+    #[arg(long)]
+    pub open: bool,
+    /// An optional path to a custom theme directory.
+    ///
+    /// This argument is meant to be used by developers of the `wdl` crates;
+    /// customizing the theme used for the generated documentation is currently
+    /// unsupported.
+    #[arg(long, value_name = "DIR")]
+    pub theme: Option<PathBuf>,
+
+    /// Install the theme if it is not already installed.
+    ///
+    /// Requires the `--theme` argument to be specified.
```

Comment:
Maybe worth mentioning that this is an `npm` operation in this help text. Without context, I don't know what it means to install a theme.